components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -573,47 +561,14 @@ std::string BraveReferralsService::BuildReferralFinalizationCheckPayload()
   return result;
 }
 
-void BraveReferralsService::FetchReferralHeaders() {
-  net::NetworkTrafficAnnotationTag traffic_annotation =
-      net::DefineNetworkTrafficAnnotation(
-        "brave_referral_headers_fetcher", R"(
-        semantics {
-          sender:
-            "Brave Referrals Service"
-          description:
-            "Fetches referral headers from Brave."
-          trigger:
-            "An update timer indicates that it's time to fetch referral headers."
-          data: "Brave referral headers."
-          destination: WEBSITE
-        }
-        policy {
-          cookies_allowed: NO
-          setting:
-            "This feature cannot be disabled by settings."
-          policy_exception_justification:
-            "Not implemented."
-        })");
-  auto resource_request = std::make_unique<network::ResourceRequest>();
-  resource_request->url =
-      GURL(BuildReferralEndpoint(kBraveReferralsHeadersPath));
```

Comment:
we can drop the constant `kBraveReferralsHeadersPath` as well

Reply:
fixed